lighthouse-core/computed/js-bundles.js

```diff
@@ -44,22 +40,24 @@ function computeGeneratedFileSizes(map, content) {
 
     const line = lines[lineNum];
     if (line === null) {
-      log.error('JSBundles', `${map.url()} mapping for line out of bounds: ${lineNum + 1}`);
-      return failureResult;
+      const error = `${map.url()} mapping for line out of bounds: ${lineNum + 1}`;
+      log.error('JSBundles', error);
+      return {error};
```

Comment:
good point, codecov, should have a test for this case :)

Reply:
missing coverage is for the "line out of bounds" case

Reply:
ah, I meant to get that w/ https://github.com/GoogleChrome/lighthouse/pull/10449/commits/d517ec5aad15b6ed8115da340b75be7cff8b5f0e , but I tested the wrong thing (column OOB instead of line OOB)